frameworks/Clojure/duct/src/hello/boundary/world_db.clj

```diff
@@ -0,0 +1,32 @@
+(ns hello.boundary.world-db
+  (:require [duct.database.sql]
+            [duct.database.mongodb.monger]
+            [clojure.java.jdbc :as jdbc]
+            [monger.collection :as mc]))
+
+(defn- query [db]
+  (first
+    (jdbc/query db ["select * from world where id = ?" (inc (rand-int 4))])))
```

Comment:
It's supposed to be a random number between 1 and 10,000 (inclusive on both ends), so I think this should be `(rand-int 10000)` instead of `(rand-int 4)`.

Reply:
Nice catch, I only had some rows in local, before I configured Docker on my machine 👍